OpenOversight/app/api/v1/api.py

```diff
@@ -0,0 +1,36 @@
+import json
+from http import HTTPMethod, HTTPStatus
+
+from flask import Blueprint, current_app
+from sqlalchemy.orm import joinedload
+
+from OpenOversight.app.models.database import Assignment, Department, Officer, db
+from OpenOversight.app.models.database_cache import (
+    get_database_cache_entry,
+    put_database_cache_entry,
+)
+from OpenOversight.app.utils.constants import KEY_DEPT_ALL_OFFICERS
+
+
+v1 = Blueprint("v1", __name__, url_prefix="/v1")
+
+
+@v1.route("/departments/<int:department_id>/officers", methods=[HTTPMethod.GET])
+def download_dept_officers(department_id: int):
```

Comment:
Using one of the formats here: https://github.com/lucyparsons/OpenOversight/issues/385#issuecomment-1606526071

Thoughts, @sea-kelp @abandoned-prototype @b-meson @AetherUnbound?